src/MessageBird/Objects/Conversation/HSMLanguage.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace MessageBird\Objects\Conversation;
+
+class HSMLanguage
```

Comment:
Could we move the HSM* Classes into a sub-namespace? So e.g. `HSMCurrency -> MessageBird\Objects\Conversations\HSM\Currency`

Reply:
thank you for your feedback, I've moved it to the HSM namespace :)